modules/basics/src/main/java/com/opengamma/basics/index/Index.java

```diff
@@ -19,6 +21,19 @@
  * All implementations of this interface must be immutable and thread-safe.
  */
 public interface Index
-    extends Named {
+    extends Named, StandardIdentifiable {
+
+  /**
+   * Returns the standard identifier of the index.
+   * <p>
+   * The standard identifier has a scheme of "OG-Index".
+   * The value is the {@linkplain #getName() name} of the index.
+   * 
+   * @return the standard identifier
+   */
+  @Override
+  public default StandardId getStandardId() {
+    return StandardId.of("OG-Index", getName());
```

Comment:
A couple of things:

* Should this be a constant somewhere?
* I prefer to look of it compared to the old UPPERCASE_UNDERSCORE schemes, but would it be better to be familiar?